apps/ios/Shared/Model/SimpleXAPI.swift

```diff
@@ -567,6 +587,12 @@ func apiLeaveGroup(groupId: Int64) async throws -> GroupInfo {
     throw r
 }
 
+func apiListMembers(groupId: Int64) -> [GroupMember] {
```

Comment:
we did discuss that this function must be async

Reply:
we only use blocking API calls on chat start I think, and in one more case where it's difficult to make UI work otherwise

Reply:
I think it's from the attempts of reading it on button press or rather sheet opening.. I didn't manage to make it async there..